src/Psy/Command/WhereamiCommand.php

```diff
@@ -100,12 +100,12 @@ protected function fileInfo()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        extract($this->fileInfo());
+        $info = $this->fileInfo();
```

Comment:
Yes, because ``$file`` and ``$line`` variables are not declared for IDE (magic).